PeerioTechnologies/peerio-mobile
# Feat welcome screen zero state

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/13890/mobile-new-welcome-screen-when-just-signed-up
https://github.com/PeerioTechnologies/peerio-icebear/pull/314
#### Testing instructions  
Create new account -> See new welcome screen.
1.
Click any tab -> App goes back to normal flow that we all know and love
Restart app -> App goes back to normal flow that we all know and love, no new welcome screen.
2.
Login to existing account, should not see welcome screen.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
